apps/server/src/schemas/jsonSchema.ts

```diff
@@ -487,9 +487,10 @@ export const $UserCreation: JSONSchema<S.UserCreation> = {
     email: { type: 'string' },
     groups: { type: 'array', items: $Ulid },
     securityTrainingCompletedAt: { type: 'integer' },
+    autoEmail: { type: 'boolean' }
```

Comment:
consider: Rename to be easier to understand (`sendAccountCreationEmail`), or at least add a comment here to explain what this represents (e.g. `// Whether to send an email to the user with their newly created account details`.